### PR TITLE
Make package filter case-insensitive.

### PR DESCRIPTION
**Pull request by mitchell-as**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-401" title="DX-401" target="_blank">BugDX-401</a>  Filtering packages in a project is case-sensitive
  </td></table>
  <br />

### CODE REVIEW

**Comment by mitchell-as** [comment]
Test failures are not due to this PR.

File: internal/runners/packages/list.go [hunk 1]
```diff
@@ -162,7 +162,7 @@ func newFilteredRequirementsTable(requirements []*gqlModel.Requirement, filter s
 
 	rows := make([]packageRow, 0, len(requirements))
 	for _, req := range requirements {
-		if !strings.Contains(req.Requirement, filter) {
+		if !strings.Contains(strings.ToLower(req.Requirement), strings.ToLower(filter)) {
```
Comment (MDrakos) [inline internal/runners/packages/list.go:165]: I think this is a good case for `strings.EqualFold`

**Review by MDrakos — CHANGES_REQUESTED** [review]

File: internal/runners/packages/list.go (hunk 1 above)
Comment (mitchell-as) [reply internal/runners/packages/list.go:165]: I think we want to have a substring search though. If you look at the ticket you will see the user expected that searching for 'flask' and 'Flask' would result in matches for any package with the 'flask-' or 'Flask-' prefix.

Comment (MDrakos) [reply internal/runners/packages/list.go:165]: Ahh I see, okay makes sense 👍

**Review by MDrakos — APPROVED** [review]

**Pull request merged**